### PR TITLE
[JSON schemas] support recursion in generating models

### PR DESCRIPTION
**Pull request by OlivierChirouze**
And remove transmission-response-no-recursion.json

### CODE REVIEW

File: mvp-spec/model/audit-log.md [hunk 1]
```diff
@@ -12,6 +12,36 @@ An Audit Log gathers the necessary data to audit a Transaction (via Transmission
     <th> Description </th>
 </tr>
 
+<tr>
```
Comment (OlivierChirouze) [inline mvp-spec/model/audit-log.md:15]: This is unrelated with this PR: the model was not up to date on `main`.

File: mvp-spec/model/post-seed-request.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
 
-# POST /v1/seed request
+# POST /paf-proxy/v1/seed request
```
Comment (OlivierChirouze) [inline mvp-spec/model/post-seed-request.md:3]: This is unrelated with this PR: the model was not up to date on `main`.

File: mvp-spec/model/post-seed-response.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
 
-# POST /v1/seed request
+# POST /paf-proxy/v1/seed response
```
Comment (OlivierChirouze) [inline mvp-spec/model/post-seed-response.md:3]: This is unrelated with this PR: the model was not up to date on `main`.

File: mvp-spec/model/transmission-request.md [hunk 4]
```diff
@@ -694,7 +694,7 @@ The base64 representation of a data signature
 
 <tr>
 <td>
-<b>receiver</b>
+receiver<br>(<i>optional</i>)
```
Comment (OlivierChirouze) [inline mvp-spec/model/transmission-request.md:697]: This is unrelated to this PR: the model was not up to date on `main`.
(same below)

File: mvp-spec/model/open-rtb-bid-response.md [hunk 5]
```diff
@@ -340,306 +340,9 @@ array
...
-
-</details>
-
-</td>
-</tr>
-
-</table>
-
-</details>
+Transmission Response (recursive)
```
Comment (OlivierChirouze) [inline mvp-spec/model/open-rtb-bid-response.md:345]: Note: I tried to use a hyperlink but this was more complicated than I expected: I leave it as a future improvement.
At least this is taking the name of the schema.

**Review by RomainLofaso — APPROVED** [review]
